documentation/endpoints/kudos.md

```diff
@@ -20,16 +21,75 @@ GET https://www.dosomething.org/api/v1/kudos
 
 ### Optional query parameters
 
+**NOTE: The following query parameters no longer work with this endpoint and will soon be deprecated or refactored!**
```

Comment:
Well, they're still passable... and results kinda vary. Leaving them in so if _used_ it's understood that shit might be a little weird in the response. I tried it and you get a response... but it's slightly deceiving in what it replies. Once I remove those URL params, I'll fully remove them from the docs.